kal_Q: 1
kal_R: 0.1